src/main/res/values/strings.xml

```diff
@@ -1181,13 +1181,13 @@
 
 	<!-- 2020-02-20 -->
 	<string name="upgrade_message">Updated to version %s</string>
-	<string name="error_toast_failed_open_external_browser">Failed to open url \"%s\" in external browser</string>
-	<string name="error_toast_no_email_apps">No email apps installed!</string>
-	<string name="bug_chooser_title">Email bug report</string>
+	<string name="error_toast_failed_open_external_browser">Could not open the URL "%s" in any web-browser</string>
```

Comment:
Preferred original, "any" could include the internal browser

Reply:
Yes, that presumably also failed?

Reply:
The string is used in cases where the external browser was explicitly requested (e.g. due to settings) and the internal browser wasn't tried.

Reply:
"External browser" is also consistent with preferences/etc, and may encompass things that aren't strictly web browsers (e.g. the YouTube app) if they are set up to handle certain links